quilt/tools/command.py

```diff
@@ -437,16 +443,16 @@ def _worker_thread():
                                 temp_file.seek(0)
 
                                 # Workaround for non-local variables in Python 2.7
-                                class ctx:
+                                class Context:
                                     compressed_read = 0
                                     original_last_update = 0
 
                                 def _progress_cb(count):
-                                    ctx.compressed_read += count
-                                    original_read = ctx.compressed_read * original_size // compressed_size
+                                    Context.compressed_read += count
+                                    original_read = Content.compressed_read * original_size // compressed_size
```

Comment:
Uh... `Context`, not `Content`?

This code is kinda impossible to unit test, unfortunately... Though I'd expect the linter to catch this.

Reply:
DONE: good catch.  sigh, pylint FTL